README.md

```diff
@@ -112,7 +112,21 @@ Requirements
   - `jenkins2_credentials_enabled` - to add credentials
      default: `True`
   - `jenkins2_credentials` -  credentials map
-
+# sonarqube configuration
+  - `jenkins2_sonarqube_enabled` - to add SonarQube configuration
+     default: `False`
```

Comment:
lets enable it by default to check by tests
or enable it in molecule

Reply:
Enabled in molecule tests.